File: After BootCamp/week3/1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        i = 0
        j = len(nums) - 1
        
        for k in range(len(nums)):
            nums[k] = (nums[k],k)
            
        nums.sort()
        
        while i <= j:
            temp = nums[i][0] + nums[j][0]
            if temp > target:
                j -= 1
                
            elif temp < target:
                i += 1
                
            elif temp == target:
                return [nums[i][1],nums[j][1]]

File: After BootCamp/week5/1-two-sum/1-two-sum.js
var twoSum = function(nums,target){
    for(var i = 0; i<nums.length;i++){
        var expected = target - nums[i];
        if(nums.includes(expected)&&nums.indexOf(expected) !== i){
            return [i,nums.indexOf(expected)];
            }
        }
    return []
}